packages/terra-clinical-onset-picker/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Unreleased
 4.5.0 - (August 14, 2019)
 ----------
 ### Changed
+* Removed age granularity from being populated for age less than a week.
```

Comment:
This must have moved into a released section while merging maser, we'll need to add it under unreleased. We'll want to place it within a `# Fixed` section as well

Reply:
Fixed here https://github.com/cerner/terra-clinical/pull/543/commits/37b811e87bebdcdb25bd349920472977e3217844.